server/model/userModel.js

```diff
@@ -3,5 +3,15 @@ module.exports={
     //TODO:remove
     test:(params)=>{
         return User.find({}).exec();
+    },
+    save:(credentials)=>{
+        const newUser = new User({
+            name: credentials.name,
```

Comment:
ok, what about name duplicates ?
Maybe better to make sure that they are **unique**

Reply:
Done
Also added 
`mongoose.set('useCreateIndex', true);`
in mongo.js file because of deprecation warning